### PR TITLE
.papr.yml: Update to Fedora 30

### PR DESCRIPTION
**Pull request by mwleeds**
We need at least GLib 2.60.1 for this PR:
https://github.com/flatpak/flatpak/pull/2797

### CODE REVIEW

File: .papr.yml [hunk 1]
```diff
@@ -6,13 +6,13 @@ branches:
...
 required: true
-context: f29-primary
+context: f30-primary
 
 # This test case wants an "unprivileged container with bubblewrap",
 # which we don't have right now; so just provision a VM and do a
 # docker --privileged run.
 host:
-  distro: fedora/29/atomic
+  distro: fedora/30/atomic
```
Comment (cgwalters) [inline .papr.yml]: This doesn't exist.  Also I don't think you need to bump it, just the container right?

Comment (mwleeds) [reply .papr.yml]: I don't know. Which container?

Comment (mwleeds) [reply .papr.yml]: Oh, I see. You're saying run an F30 container on F29. Yeah that should work.

**Comment by mwleeds** [comment]
Not sure what the test failure means:
```
+ yum -y install @buildsys-build
Last metadata expiration check: 0:02:58 ago on Fri Sep  6 17:01:57 2019.
Error: 
 Problem: problem with installed package fedora-release-container-30-5.noarch
  - package fedora-release-container-30-5.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-5.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - conflicting requests
  - package fedora-release-container-30-5.noarch conflicts with system-release provided by fedora-release-30-5.noarch
  - package fedora-release-30-5.noarch conflicts with system-release provided by fedora-release-container-30-5.noarch
  - package fedora-release-30-5.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-5.noarch
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
### EXITED WITH CODE 1 AFTER 221s
```

**Comment by cgwalters** [comment]
See https://github.com/ostreedev/ostree/commit/d85366d52a3c837fc11d93043f023d37b6b228aa#diff-762b1653e39c3f06de4394809133778dR40
(Yeah it's lame)

**Comment by mwleeds** [comment]
Thanks! Looks like it still fails, but they're unit test failures, so that's progress.

**Comment by alexlarsson** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**